internal/services/signalr/signalr_service_resource.go

```diff
@@ -286,7 +286,10 @@ func resourceArmSignalRServiceCreate(d *pluginsdk.ResourceData, meta interface{}
 	} else {
 		expandedFeatures = append(expandedFeatures, signalRFeature(signalr.FeatureFlagsEnableConnectivityLogs, strconv.FormatBool(connectivityLogsEnabled)))
 		expandedFeatures = append(expandedFeatures, signalRFeature(signalr.FeatureFlagsEnableMessagingLogs, strconv.FormatBool(messagingLogsEnabled)))
-		expandedFeatures = append(expandedFeatures, signalRFeature(signalr.FeatureFlagsServiceMode, serviceMode))
+
+		if serviceMode != "" {
```

Comment:
shouldn't this be defaulting this if it's not specified?

Reply:
This is computed in the schema at the minute until the old features block can be removed in 3.0 so I can't set a default there yet, but I could change it to be this if it makes more sense:

```
if serviceMode == "" {
  serviceMode!= "Default" 
}
```

Reply:
that'd work 👍